mjcf_to_sdformat/mjcf_to_sdformat/converters/geometry.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module to convert MJCF geoms to SDFormat Collision/Visual geometries"""
+
+from ignition.math import Vector2d, Vector3d
+
+import sdformat as sdf
+import sdformat_mjcf_utils.sdf_utils as su
+
+VISUAL_NUMBER = 0
+COLLISION_NUMBER = 0
+
+
+def add_mjcf_geometry_to_sdf(geom):
+    """
+    Converts an MJCF geom to a SDFormat geometry.
+
+    :param mjcf.Element geom: The MJCF geom
+    :return: The newly created SDFormat geometry.
+    :rtype: sdf.Geometry
+    """
+    sdf_geometry = sdf.Geometry()
+    if geom.type == "box":
+        box = sdf.Box()
+        box.set_size(su.list_to_vec3d(geom.size) * 2)
+        sdf_geometry.set_box_shape(box)
+        sdf_geometry.set_type(sdf.Geometry.GeometryType.BOX)
+    elif geom.type == "capsule":
+        capsule = sdf.Capsule()
+        capsule.set_radius(geom.size[0])
+        if geom.fromto is None:
+            capsule.set_length(geom.size[1] * 2)
+        else:
+            v1 = Vector3d(geom.fromto[0], geom.fromto[1], geom.fromto[2])
```

Comment:
When using `fromto`, it's possible that we have to adjust the pose of the 
visual/collision to match the longitudinal axis of the capsule/cylinder and its 
position. For example, a `fromto="0 0 0 1 0 0"` has the longitudinal axis along 
the x-axis, so we'll have to rotate the SDFormat capsule/cylinder by 90 
degrees, but a `fromto="0 0 0 0 0 1"` has its axis along the z-axis, so we 
don't need any adjustment on the rotation. The same applies for the position, which 
is calculated as the middle of the two points given in the `fromto`.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/26/commits/91ba48d7b80840dbed92333926e35c61c9600d24